unittests/eosio.system_tests.cpp

```diff
@@ -30,126 +30,126 @@ bool within_one(int64_t a, int64_t b) { return within_error(a, b, 1); }
 
 BOOST_AUTO_TEST_SUITE(eosio_system_part1_tests)
 
-BOOST_FIXTURE_TEST_CASE( buysell, eosio_system_tester ) try {
+BOOST_AUTO_TEST_CASE_TEMPLATE( buysell, T, eosio_system_testers ) try {
```

Comment:
You could use `BOOST_FIXTURE_TEST_CASE_TEMPLATE( buysell, T, eosio_system_testers, T )`, but this works too. You'd still have to prefix `transfer`, etc... with `T::`.